configs/am67-sk.cpp

```diff
@@ -0,0 +1,85 @@
+/* Configuration file for AM67 SK*/
+
+#include <iostream>
+#include "backend/includes/common.h"
+#include "backend/includes/live_camera.h"
+#include "backend/includes/run_cmd.h"
+#include "backend/includes/settings.h"
+#include "backend/includes/gpu_performance.h"
+#include "backend/includes/benchmarks.h"
+
+#define PLATFORM "am67-sk"
+using namespace std;
+QString platform = "am67-sk";
+QString wallpaper = "file:///opt/ti-apps-launcher/assets/am6x_oob_demo_home_image.png";
+
+int include_powerbuttons_count = 2;
+power_actions include_powerbuttons[] = {
+    {
+        .name = "Shutdown",
+        .command = "shutdown now",
+        .icon_source = "file:///opt/ti-apps-launcher/assets/shutdown.png",
+    },
+    {
+        .name = "Reboot",
+        .command = "reboot",
+        .icon_source = "file:///opt/ti-apps-launcher/assets/reboot.png",
+    }
+};
+
+int include_apps_count = 6;
+app_info include_apps[] = {
+    {
+        .qml_source = "industrial_control.qml",
+        .name = "Industrial HMI",
+        .icon_source = "file:///opt/ti-apps-launcher/assets/hmi.png"
+    },
+    {
+        .qml_source = "live_camera.qml",
+        .name = "Live Camera",
+        .icon_source = "file:///opt/ti-apps-launcher/assets/camera.png"
+    },
+    {
+        .qml_source = "benchmarks_jacinto.qml",
+        .name = "Benchmarks",
+        .icon_source = "file:///opt/ti-apps-launcher/assets/benchmarks.png"
+    },
+    {
+        .qml_source = "gpu_performance.qml",
+        .name = "GPU Performance",
+        .icon_source = "file:///opt/ti-apps-launcher/assets/gpu_performance.png"
+    },
+    {
+        .qml_source = "seva_store.qml",
+        .name = "Seva Store",
+        .icon_source = "file:///opt/ti-apps-launcher/assets/seva_store.png"
+    },
+    {
+        .qml_source = "settings.qml",
+        .name = "Settings",
+        .icon_source = "file:///opt/ti-apps-launcher/assets/settings.png"
+    }
+};
+
+Settings settings;
+LiveCamera live_camera;
+Benchmarks benchmarks;
+Gpu_performance gpuperformance;
+
+RunCmd *seva_store = new RunCmd(QStringLiteral("su weston -c \"chromium http://localhost:8000/#/\""));
+RunCmd *sdk_datasheet = new RunCmd(QStringLiteral("docker run -v /run/user/1000/:/tmp/ -i --env http_proxy --env https_proxy --env no_proxy --env XDG_RUNTIME_DIR=/tmp/ --env WAYLAND_DISPLAY=wayland-1 -u user ghcr.io/texasinstruments/seva-browser:v1.0.0 https://software-dl.ti.com/jacinto7/esd/processor-sdk-linux-am68/latest/exports/docs/devices/J7_Family/linux/Release_Specific_Performance_Guide.html"));
```

Comment:
As Firefox is removed from filesystem [1], this command would pull the docker image from ghcr.io & then launch. This will take atleast 3-5 mins on good bandwidth. Are we not planning to add Chromium on AM67?

[1]: https://git.ti.com/cgit/ti-sdk-linux/meta-tisdk/commit/?h=kirkstone&id=70dce5ff45bf02fd8fdcf8ef9d7fd55f5e1aa6b0

Reply:
I actually missed that...
So this is a copy from m68/am69 where we were doing this.... 
https://github.com/TexasInstruments/ti-apps-launcher/commit/291f4a0b9a1e0c60f2cf9246b82b98ded8bac123#diff-0a846b821dd94bac501975c94b83911254110c349cb82f20e472af1ebc4e91c3R71

Maybe this will need to change all across am67/am68/am69. Can you help with what it should be instead?
Should it now be the below instead?
su weston -c \"chromium https://software-dl.ti.com/jacinto7/esd/processor-sdk-linux-am68/latest/exports/docs/devices/J7_Family/linux/Release_Specific_Performance_Guide.html\"

Reply:
@aniket-l  Yeah, looks like this has to be updated for all J7 platforms.

Following command should work,
```sh
RunCmd *sdk_datasheet = new RunCmd(QStringLiteral("su weston -c \"chromium https://software-dl.ti.com/jacinto7/esd/processor-sdk-linux-am68/latest/exports/docs/devices/J7_Family/linux/Release_Specific_Performance_Guide.html\""));
```
Maybe you can verify it once by building ti-apps-launcher locally before updating the PR.

Reply:
@cshilwant I have updated the PR by adding an additional commit to add chromium app to am68, am69. Followed by the support for am67 rebased to that commit (with the chromium support). I have also build-tested the branch locally